### PR TITLE
Use `.mjs` extension

### PR DESCRIPTION
**Pull request by jeffsee55**

### CODE REVIEW

**Comment by danclaytondev** [comment]
**This PR fixes TinaCMS with Astro for me on Astro 2.7.2 :)** 

I was having significant issues using TinaCMS (v1.5.16) with Astro (v2.7.2), which are the same as described in #3764. Installing the tagged release for this PR allows me to use Tina now!

Until this is merged I will be using `tinacms@fix-es-extension` and `@tinacms/cli@fix-es-extension`.

File: experimental-examples/tina-cloud-starter/content/posts/post/post/post/post/thats-not-good.mdx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (kldavis4) [inline experimental-examples/tina-cloud-starter/content/posts/post/post/post/post/thats-not-good.mdx:1]: drop these test posts?

File: packages/tinacms/src/admin/pages/CollectionCreatePage.tsx [hunk 2]
```diff
@@ -197,14 +197,22 @@ export const RenderForm = ({
             schemaCollection.path +
             folderName +
             `/${values?.values?.filename}.${schemaCollection.format || 'md'}`
+          console.log({
```
Comment (kldavis4) [inline packages/tinacms/src/admin/pages/CollectionCreatePage.tsx:200]: delete?

**Review by kldavis4 — COMMENTED** [review]
@jeffsee55 tested this and I'm not seeing any errors on a demo project outside the monorepo. Just a couple minor cleanup suggestions.

**Comment by kldavis4** [comment]
also - Is the slugify function stuff in here related work? does it need a changeset?

**Review by kldavis4 — APPROVED** [review]

**Pull request merged**